deepchem/models/torch_models/tests/test_highway.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+try:
+    import torch
+    import numpy as np
+    has_torch = True
+except ModuleNotFoundError:
+    has_torch = False
+    pass
+from deepchem.models.torch_models.layers import Highway
+
+
+@pytest.mark.torch
+def test_highway_shape_no_params():
+    width = 5
+    batch_size = 2
+    torch.manual_seed(42)
+
+    input = torch.rand(batch_size, width, dtype=torch.float32)
+    layer = Highway(layer_shape=[width, 2])
+    result = layer(input)
+    assert result.shape == (batch_size, width)
+    assert len(list(layer.parameters())) == 0
+
+    result2 = layer(input)
+    assert torch.allclose(result, result2)
+
+
+@pytest.mark.torch
+def test_highway_layer_non_deterministic_output():
```

Comment:
`RuntimeError: The size of tensor a (5) must match the size of tensor b (2) at non-singleton dimension 1` Error in the tests